mitiq/_version.py

```diff
@@ -1,24 +1 @@
-# Copyright (C) 2020 Unitary Fund
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-
-"""Reads in version information.
-
-Note: This file will be overwritten by the packaging process."""
-import os
-
-directory_of_this_file = os.path.dirname(os.path.abspath(__file__))
-
-with open(f"{directory_of_this_file}/../VERSION.txt", "r") as f:
-    __version__ = f.read().strip()
+__version__ = "0.5.0dev"
```

Comment:
Was this intentional? I don't think `VERSION.txt` is removed in this PR so I'm not sure.

Reply:
Nope this should not have changed, I'll fix it.